lib/game.rb

```diff
@@ -13,14 +13,15 @@ def over?
   end
 
   def play
+    greet
     while !over?
       play_turn
     end
+    display_board
   end
 
   def play_turn
-    position = get_position
-    board.add_move(position, current_player.mark)
+    add_to_board(get_position, current_player.mark)
```

Comment:
I like that you got rid of the temp variable. This reads very nice, with the entire method at the same level of abstraction.